classes.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class HistoryRepository
+{
+    protected $DS = DIRECTORY_SEPARATOR;
+    protected $fileName;
+
+    public function __construct($dirname, $fileName)
```

Comment:
these fields shouln't be set via cinstructor, let's fill it directly inside the class

```suggestion
    public function __construct()
```